frontend/src/metabase/selectors/app.ts

```diff
@@ -66,18 +136,13 @@ export const getIsAppBarVisible = createSelector(
     isAdminApp,
     isEditingDashboard,
     isEmbedded,
-    embedOptions,
+    isEmbeddedAppBarVisible,
   ) => {
     const isFullscreen = hash.includes("fullscreen");
-    const allEmbeddedAppBarElementsHidden =
```

Comment:
this is moved to https://github.com/metabase/metabase/pull/27391/files#diff-27c349cf6072ae10fa01d3ecd7d3607c7629b096f7addadc169a78c78318785aR98